### ISSUE TITLE
Endpoints not expanding when clicked

### ISSUE THREAD

**Issue by BronOS** [swagger-api/swagger-ui#6254]
The endpoints are not expanding/opening when clicking on it.
OpenAPI document version: 3.0.0
OpenAPI Format: YAML, JSON (spec has been checked with swagger editor)
JS file: //unpkg.com/swagger-ui-dist@3/swagger-ui-bundle.js
HTML file:

```html
<html>
<head>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/swagger-ui-dist@3.17.0/swagger-ui.css">
    <script src="//unpkg.com/swagger-ui-dist@3/swagger-ui-bundle.js"></script>
    <script>

        function render() {
            var ui = SwaggerUIBundle({
                url: `swagger.yaml`,
                dom_id: '#swagger-ui',
                presets: [
                    SwaggerUIBundle.presets.apis,
                    SwaggerUIBundle.SwaggerUIStandalonePreset
                ]
            });
        }

    </script>
</head>

<body onload="render()">
<div id="swagger-ui"></div>
</body>
</html>
```

**Comment by hkosova** [comment]
@BronOS is this issue the same as #6249?

**Comment by BronOS** [comment]
Behavior is the same, I found it at this morning as end user... The title of ticket is confusing me - "3.30.1 breaks path tags» that’s why I didn’t found it before send to you…  I have no idea what is «path tags» the same as 3.30.1 as well ;) also it doesn’t belong to MacOS and chrome browser, it can be reproduced on any platforms and browsers, for example I reproduced it on Mac/Win/Ubuntu + safari/chrome/firefox

**Comment by BronOS** [comment]
you can close this as a duplicate, but pls change the title of original ticker before

**Comment by SergeyMell** [comment]
Same issue for me. Found today. Previously used `3.28.0` version - worked ok
macOS Cathalina 10.15.4 (Chrome 84.0.4147.89)

**Comment by soundstripe** [comment]
Same here. Authorization button also missing.

**Comment by SergeyMell** [comment]
As a workaround one may just add `"swagger-ui-dist": "3.28.0",` directly to `package.json` to override internal dependency.

**Comment by BronOS** [comment]
is there any chance to use 3.30.0 version with "unpkg"?

**Comment by hkosova** [comment]
@BronOS
> is there any chance to use 3.30.0 version with "unpkg"?

https://unpkg.com/swagger-ui-dist@3.30.0/swagger-ui-bundle.js

**Comment by dpgaspar** [comment]
same here!

**Comment by flashgorman-chorus** [comment]
same

**Comment by ten-lac** [comment]
same here.

**Comment by dieg0moraes** [comment]
same issue here

**Comment by tim-lai** [comment]
Duplicate of #6249 , which is now resolved with a new release build [v3.30.2](https://github.com/swagger-api/swagger-ui/releases/tag/v3.30.2). Closing issue.
